### ISSUE TITLE
About the hyperparameter

### ISSUE THREAD

**Issue by JinYujie99** [shengliu66/SOP#3]
Dear Liu,

Thanks for your nice work! The implementation is slightly different from the Algorithm1. It seems that the learning rate for "u" is set to constant 1 and the discrepant learning rate for "v" is achieved by the hyperparameter "ratio_reg". 

What should I do if I want to tune the hyperparameter "alpha_u" as in Table 5 in your paper?

Best

**Comment by penghui-yang** [comment]
'u' uses CE as its loss function and 'v' uses MSE. Those two loss functions have different gradients. That is the reason why they need different alpha.
